ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -445,8 +447,12 @@ private[state] object Conversions {
   def decodeTransactionRejectionEntry(
       entry: DamlTransactionRejectionEntry
   ): Option[FinalReason] = {
-    def buildStatus(code: Code, message: String) = {
-      Status.of(code.value, message, Seq.empty)
+    def buildStatus(code: Code, message: String, metadata: Map[String, String]) = {
```

Comment:
@miklos-da, `definite_answer` is not something that we'd like to store, right? So, we can just convert it while decoding.

Reply:
I don't fully understand this comment as I don't know where this should not be stored. Whether something is a definite answer should be decided by the committer in kvutils. Accordingly, I'd expect that the participant needs to store whether the committer has marked something as `definite_answer` or not. If this comment is completely out of place, just ignore it.

Reply:
@miklos-da recommended adding the `definite_answer` to the proto according to: https://github.com/digital-asset/daml/pull/9499/files#diff-a5535746ffadbcb0c4b32562d134d1de6f87e488eb96aefd72cea1ed1a0748e8R269, as in the future it won't be possible to determine whether a response is definite or not on the participant-side (here). Therefore it is going to be persisted on the ledger and just populated in the `Status` here, so we have everything in place.